Channels/Integration/01c_Salesforce.md

```diff
@@ -0,0 +1,487 @@
+    The *Select and Configure Step* side bar is displayed on the right of the workspace.
+
+7. In the *Select and Configure Step* side bar, the following values must be configured: 
+
+    | Field                    | Value     |
+    |--------------------------|-------------------------|
+    | Context                  | Organization            |
+    | ID                       | ImportInventory lists   |
+    | Description              | -                       |
+    | WorkingFolder            | import/inventory        |   
+    | FileNamePattern          | ac_.*_inventory\\.xml   | 
+    | NofilesFoundHandling*    | NO_FILES_FOUND          |
+    | ImportMode*              | Merge                   |  
+    | ImportFailedHandling*    | WARN                    |  
+    | AfterImportFileHandling* | Archive                 |
+    | ArchiveFolder            | archive/inventory       |  
+
+[comment]: <> (Scope: Organization - Wo einzustellen? Relevant hier? Job FLow -> Was ist? Muss man ein Job flow erstellen, bevor man ein Step erstellt?)
+
+8. Click on [Assign].  
+    The job step has been configured.
+
+9. Click again the [New Job] button in the upper right corner to create the data import job.  
+    The *New Job* window is displayed on the right side of the workspace.
+
+10. Enter the job name "MSN-SiteDataImport" in the *ID* field and, if desired, a job description in the *Description* field, and click the [Create] button.  
+    The new job has been created. The job page is displayed.
+
+    The MSN-SiteDataImport imports different types of data and is therefore a multi-step job. Repeat steps **6** to **8** to create and configure the following
+    job steps:
+
+
+    - StandardProductsImport (Scope: Organization)  
+
+        | Field                    | Value     |
+        |--------------------------|-------------------------|
+        | Context                  | Organization            |
+        | ID                       | StandardProductsImport  |
+        | Description              | -                       |
+        | WorkingFolder            | import/catalogs         |   
+        | FileNamePattern          | ac_.\*_catalog\_.*\\.xml| 
+        | NofilesFoundHandling*    | NO_FILES_FOUND          |
+        | ImportMode*              | Merge                   |  
+        | ImportFailedHandling*    | WARN                    |  
+        | AfterImportFileHandling* | Archive                 |
+        | ArchiveFolder            | archive/catalogs        |  
+
+        > [Info] The Import/catalogs working folder must be previously created in WebDAV. Otherwise, the data cannot be imported. For detailed information, see [Using WebDAV](https://documentation.b2c.commercecloud.salesforce.com/DOC1/index.jsp?topic=%2Fcom.demandware.dochelp%2Fcontent%2Fb2c_commerce%2Ftopics%2Fimport_export%2Fb2c_using_web_dav.html).
+
+    - BundleProductsImport (Scope: Organization)  
```

Comment:
13. Click the [] (Add) button below the first flow.
   A new flow has been added below the first flow. 
   
14. Select the **Organization** option as a scope for the flow.

15. Click the [Configure a step] button in the flow box and select the *BundleProductsImport* step.
   The *Select and Configure Step* side bar is displayed on the right side of the page.
   
16. Configure the following values in the corresponding fields of the *Select and Configure Step* side bar for the *BundleProductsImport* step: